lib/sorbet-rails/typed_params.rb

```diff
@@ -0,0 +1,17 @@
+# typed: true
```

Comment:
I thought you'd have to make this file `# typed: ignore` or sorbet will know that `[]` is overridden and doesn't respect the sig

Reply:
I think it works since `define_singleton_method` changes the runtime behaviors so it does not affect the static type-checking.

Reply:
That's not enough because sorbet will generate dynamic rbi that will produce `self.[]` method. The way to go around that is to tell sorbet to ignore the file.

Reply:
Actually, I don't remember why I did all the measures I did to hide this method. We just need to test very thoroughly, esp by running `srb rbi init` in a small project after `sorbet-rails` is installed.